generative/inferers/inferer.py

```diff
@@ -381,15 +381,18 @@ def __call__(
         if self.ldm_latent_shape is not None:
             latent = self.ldm_resizer(latent)
 
-        call = partial(super().__call__, seg = seg) if \
-            isinstance(diffusion_model, SPADEDiffusionModelUNet) else super().__call__
+        call = (
+            partial(super().__call__, seg=seg)
+            if isinstance(diffusion_model, SPADEDiffusionModelUNet)
+            else super().__call__
+        )
```

Comment:
```suggestion
        call = super().__call__
        if isinstance(diffusion_model, SPADEDiffusionModelUNet):
            call = partial(call, seg=seg)
```

I feel once a if-else expression is multi-line you should use an if statement instead.